ov/profilesv3.go

```diff
@@ -57,3 +58,54 @@ type OSCustomAttribute struct {
 	Name  string `json:"name,omitempty"`  // "name": "custom attribute 1",
 	Value string `json:"value,omitempty"` // "value": "custom attribute value"
 }
+
+// create profile from template
+func (c *OVClient) CreateProfileFromTemplateWithI3S(name string, template ServerProfile, blade ServerHardware, osDeploymentPlan OSDeploymentPlan, deploymentSettings map[string]string) error {
+	log.Debugf("TEMPLATE : %+v\n", template)
+	var (
+		new_template ServerProfile
+		err          error
+	)
+
+	//GET on /rest/server-profile-templates/{id}new-profile
+	if c.IsProfileTemplates() {
+		log.Debugf("getting profile by URI %+v, v2", template.URI)
+		new_template, err = c.GetProfileByURI(template.URI)
+		if err != nil {
+			return err
+		}
+		new_template.Type = "ServerProfileV6"
+		new_template.ServerProfileTemplateURI = template.URI // create relationship
+		log.Debugf("new_template -> %+v", new_template)
+	} else {
```

Comment:
I don't think we can use Image streamer with v1, so we should just error I think.